/usr/lib/python2.7/site-packages was somehow hardcoded in
a the source coude but on Debian it should be:
/usr/lib/python2.7/dist-packages
Fixing it.
--- a/vdsm-tool/vdsm-tool
+++ b/vdsm-tool/vdsm-tool
@@ -34,7 +34,7 @@
 # upgrade we need to explicit import vdsm.tool from /usr/lib. Otherwise
 # all tool's verbs are loaded from the old vdsm code during upgrade.
 lib_dir = os.path.join(os.path.dirname(os.__file__).
-                       replace('lib64', 'lib'), 'site-packages')
+                       replace('lib64', 'lib'), 'dist-packages')
 vdsm = imp.load_module('vdsm', *imp.find_module('vdsm', [lib_dir]))
 vdsm.tool = imp.load_module('vdsm.tool',
                             *imp.find_module('tool', vdsm.__path__))
